### PR TITLE
Updated fr content for the homepage

### PR DESCRIPTION
**Pull request by khalidelaggan**

### CODE REVIEW

File: i18n/fr.yaml [hunk 1]
```diff
@@ -121,38 +121,38 @@ spa-research:
...
-  other: "Votre courriel"
+  other: "Adresse courriel"
 subscribe:
-  other: "Abonnez-vous"
+  other: "S’inscrire"
 view-privacy-notice:
-  other: "Voir l'avis de confidentialité"
+  other: "Voir l’Avis de confidentialité"
 privacy-notice:
   other: "Énoncé de confidentialité"
```
Comment (EliseKa) [inline i18n/fr.yaml]: "Avis de confidentialité"

File: i18n/fr.yaml [hunk 2]
```diff
@@ -121,38 +121,38 @@ spa-research:
...
+  other: "Tout renseignement personnel qui pourra être recueilli est décrit dans le Fichier de renseignement personnel ordinaire intitulé <a href="http://www.infosource.gc.ca/emp/emp03-fra.asp#psu938" target="_blank">Activités de sensibilisation, POU 938</a> qui figure sur la page Web du SCT intitulée <a href="https://www.canada.ca/fr/secretariat-conseil-tresor/services/acces-information-protection-reseignements-personnels/acces-information/renseignements-programmes-fonds-renseignements/fichiers-renseignements-personnels-ordinaires.html" target="_blank">Renseignements sur les programmes et les fonds de renseignements</a>."
```
Comment (EliseKa) [inline i18n/fr.yaml]: The right url is the one I added in the Google doc :)

**Comment by khalidelaggan** [comment]
[Editing French content on website homepage.](https://trello.com/c/kAHtj2FG/80-editing-french-content-on-website-homepage)

File: i18n/fr.yaml [hunk 3]
```diff
@@ -91,11 +91,11 @@ rejoindre-notre-equipe:
...
   other: "Ce que nous faisons"
 contact-us:
-  other: "Rejoindre la conversation"
+  other: "Participez à la conversation"
 featured-post:
-  other: "Articles de blog récents"
+  other: "Récents billets de blogue"
 more-posts:
-  other: "Voir tout"
+  other: "Voir tous les produits"
```
Comment (EliseKa) [inline i18n/fr.yaml]: It seems that it's the same button for Blog and Products. But for blog posts it should say "Voir tous les billets" and for products it should say "Voir tous les produits"

File: i18n/fr.yaml (hunk 1 above)
Comment (EliseKa) [inline i18n/fr.yaml]: line 136: title should be "Avis de confidentialité", so it is the same words as in line 134.

**Comment by khalidelaggan** [comment]
Looks good to me now

**Pull request merged**